addons/loyalty/models/loyalty_program.py

```diff
@@ -126,7 +182,7 @@ def _program_type_default_values(self):
                 'applies_on': 'current',
                 'trigger': 'auto',
                 'portal_visible': False,
-                'portal_point_name': _('Points'),
+                'portal_point_name': _('Promo point(s)'),
```

Comment:
We have to choose between putting it in plural all the time or putting it like this. (for the (s))